department-of-veterans-affairs/va.gov-cms
# VACMS-8298 VAMC facility accordion audit

## Description

Closes #8298 

## Testing done
Viewed the accordion audit and verified the new audit lives under facilities then tested the filters and verified the results. 

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/158467274-bb052066-3c2e-4d65-9e85-a1cce21bbb62.png)


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As cms editor I can
1. Navigate to the accordion audit (`admin/content/facilities/accordion-audit`) and
2. Verify the view is a child of existing "facilities" view
3. test the accordion label filter by entering in 'parking' and getting results and
4. test the title filter by entering in 'city' and getting results
5. Then validate Acceptance Criteria from issue
  - [x] display machine name should not be default
  - [x] limit to 100 per page, paginated
  - [x] show result count in header
  - [x] default sort is last update (newest at top)
  - [x] Menu item: Add as a child to the Facilities view
  - [x] Relationship "field_location_services" to its paragraph
  - [ ] Fields
    - [x] Title (sortable) 
    - [x] Edit button  
    - [x] Moderation State
    - [x] Section (field_administration) (sortable)
    - [x] Last updated (date time and by user) (sortable, default sort, DESC)
    - [x] Accordion header field from `field_location_services` paragraphs
    - [x] Content type: VAMC facility (not exposed)
    - [x] "Accordion label contains" searching text contains on Accordion header
    - [x] Facility Title (exposed)
    - [x] moderation state (exposed)
    - [x] section (exposed) (workbench access section; single select, show children)
  - [x] Reset button exists
  - [x] Apply button says 'filter'
  - [x] CSV export that respects the filters.

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
